google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -902,6 +903,11 @@ def _convert_cloudsqlinstance(self, cloudsqlinstance):
         data = cloudsqlinstance.get_data()
         parent, full_res_name, type_name = self._full_resource_name(
             cloudsqlinstance)
+        parent_key = get_key_from_type_name(parent.type_name)
+        resource_identifier = '{}:{}'.format(parent_key,
```

Comment:
Will we generate this `resource_identifier` elsewhere?  Would it be better to be in the `util` method, instead of the `get_key_from_type_name()`

Reply:
resource_identifier will not be used elsewhere, this is a one off fix for cloudsql instance.

Reply:
Got it, thanks!  LGTM